### ISSUE TITLE
No suitable method found for w(String)

### ISSUE THREAD

**Issue by amq** [heap/react-native-heap#334]
Getting this error with the android build:

```
> Task :heap_react-native-heap:compileReleaseJavaWithJavac FAILED
/home/runner/work/reactnative-app/reactnative-app/node_modules/@heap/react-native-heap/android/src/main/java/com/heapanalytics/reactnative/RNHeapLibraryModule.java:86: error: no suitable method found for w(String)
        Log.w("Property objects must be flattened before being sent across the JS bridge. If you get this warning please inspect for non-flattenable objects being sent to Heap");
           ^
    method Log.w(String,String) is not applicable
      (actual and formal argument lists differ in length)
    method Log.w(String,String,Throwable) is not applicable
      (actual and formal argument lists differ in length)
    method Log.w(String,Throwable) is not applicable
      (actual and formal argument lists differ in length)
1 error
```

Seems like this
```
Log.w("Property objects must be flattened before being sent across the JS bridge. If you get this warning please inspect for non-flattenable objects being sent to Heap");
```
Should be:
```
Log.w("RNHeapLibraryModule", "Property objects must be flattened before being sent across the JS bridge. If you get this warning please inspect for non-flattenable objects being sent to Heap");
```

Should I open a PR?

**Comment by defnotsuhas** [comment]
Have the same issue, Commenting for further Updates.

**Comment by TheRogue76** [comment]
I added a PR. https://github.com/heap/react-native-heap/pull/335. Don't know what further steps need to be taken for the code to be merged

**Comment by gregbrinker** [comment]
@TheRogue76 what is the "suggested fix" you're referring to?

I just made a patch package with `Log.w("RNHeapLibraryModule", ...)` and it works fine now. Not sure what all the other changes were in that PR though.

**Comment by bnickel** [comment]
This has been fixed in #349 and will be released today.
